test/res.cookie.js

```diff
@@ -115,6 +115,19 @@ describe('res', function(){
         .get('/')
         .expect(200, optionsCopy, done)
       })
+
+      it('should throw an error when invalid maxAge passed', function(done){
```

Comment:
Please format new added code to the StandardJS style (see https://github.com/expressjs/express/blob/master/Collaborator-Guide.md#prs-and-code-contributions)

Reply:
@dougwilson, I didn't want to update the code that I added here because it's in the same format with the rest of the existing code .. If you want me to update it anyway, just tell me ..

Reply:
Yes, please update it :) it is a rule in our collaborator guide to follow in code changes.